### ISSUE TITLE
_.query broken

### ISSUE THREAD

**Issue by twashing** [dvv/underscore-data#1]
I think the tool may be broken. Even a small change to the first example query, returns the original object. This should return an empty result. 

``` javascript
_.query([{a:1,b:2},{a:3,b:2}], '(a=10)')
[Object, Object ]
```

I'm using the latest github version of underscore-data.js. And my browser is Chrome 17.0.963.79.

**Comment by twashing** [comment]
There's also a an open question [on StackOverflow](http://stackoverflow.com/questions/9073828/howto-use-underscore-data-formerly-underscore-query/9805392#9805392).

**Comment by dvv** [comment]
ACK. Thank you for report.

**Comment by janeklb** [comment]
It looks like this has something to do with the enclosing brackets: `'(a=10)'`. If the query is specified as `'a=10'`, it correctly returns an empty array.

**Comment by twashing** [comment]
Fair enough. Then how would I search for something in an array. This query works on a nested object structure:

``` javascript
_.query([{a:1,foo:{bar:'baz'}},{a:3,b:2}], 'foo/bar=re:ba')
```

But I can't get the following query to work on this data structure. 

``` javascript
pc = { 
  columns: [
    {   
      title: "3column-left",
      text: "text",
      image: null
    },   
    {   
      title: "thing",
    }   
  ]
}
_.query(pc, 'columns/title', '3column-left')
```
